src/components/Calculator.js

```diff
@@ -1,33 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Display from './Display';
 import './style.css';
+import calculate from '../logic/calculate';
```

Comment:
The Calculator component imports the calculate.js file from the logic directory 📝 Well done!